keps/sig-network/2438-dual-stack-apiserver/README.md

```diff
@@ -190,6 +194,25 @@ Needs to be figured out, but probably:
 <<[/UNRESOLVED]>>
 ```
 
+### kube-apiserver internals
+
+Go's network-serving methods don't allow a single HTTP server to
+listen on two different addresses. This means that either:
+
+  1. We have to duplicate a bunch of stuff in the apiserver to have
```

Comment:
has anyone from api-machinery weighed in on this?

Reply:
No. I was thinking of updating this to say that for 1.24/alpha we will _only_ support the case of dual-stack apiserver on `0.0.0.0` / `::` (which can be done without major apiserver rewrites), and then supporting dual explicit `--bind-address` would come later)

Reply:
:+1: